ui/components/component-library/avatar-favicon/avatar-favicon.js

```diff
@@ -28,17 +29,21 @@ export const AvatarFavicon = ({
       display={DISPLAY.FLEX}
       alignItems={ALIGN_ITEMS.CENTER}
       justifyContent={JUSTIFY_CONTENT.CENTER}
-      className={classnames('avatar-favicon', className)}
+      className={classnames('mm-avatar-favicon', className)}
       {...{ borderColor, ...props }}
     >
-      {imageSource ? (
-        <img className="avatar-favicon__image" src={imageSource} alt={imgAlt} />
+      {src ? (
+        <img
+          className="mm-avatar-favicon__image"
+          src={src}
+          alt={`${name} logo` || 'dapp logo'}
```

Comment:
If we make name a required prop this can just be 
```suggestion
          alt={`${name} logo`}
```